misk-grpc-tests/src/main/java/routeguide/RouteGuide.kt

```diff
@@ -0,0 +1,41 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: routeguide/RouteGuideProto.proto
+package routeguide
+
+import com.squareup.wire.Service
+import com.squareup.wire.WireRpc
+import kotlin.Pair
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.channels.SendChannel
+
+// TODO(jwilson): generate this and remove it from git.
```

Comment:
This is already generated code though?

Reply:
Wasn't getting emitted for us. Maybe fixed beyond alpha01?